.github/workflows/relevancy-e2e-test.yaml

```diff
@@ -32,7 +32,15 @@ jobs:
               scan_compliance_score,
               vulnerability_scanning_cve_exceptions,
               vulnerability_scanning_test_public_registry_connectivity_by_backend,
-              vulnerability_scanning_test_public_registry_connectivity_excluded_by_backend
+              vulnerability_scanning_test_public_registry_connectivity_excluded_by_backend,
+              relevantCVEs,
```

Comment:
@rcohencyberarmor Please advise regarding the correct tests that should run when the storage is installed and the nodeAgent is not.